lib/checkMergeConflicts.js

```diff
@@ -1,54 +1,85 @@
-    for (var label in labels) {
-      if (labels[label].name === mergeConflictLabel[0]) {
-        hasMergeConflictLabel = true;
-        break;
-      }
-    }
+    pullRequest = pullRequestPromiseObj.data;
+  }
 
-    isMergeable = pullRequestDetails.mergeable;
-
-    if (hasMergeConflictLabel === false && isMergeable === false) {
-      userName = pullRequestDetails.user.login;
-      var linkText = 'link';
-      var linkResult = linkText.link(
-        'https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/');
-      var params = context.repo({
-        number: pullRequestNumber,
-        body: 'Hi @' + userName +
-            '. Due to recent changes in the "develop" branch, ' +
-            'this PR now has a merge conflict. ' +
-            'Please follow this ' + linkResult +
-            ' if you need help resolving the conflict, ' +
-            'so that the PR can be merged. Thanks!'});
-      labelPromiseObj = await context.github.issues.addLabels(context.repo({
-        number: pullRequestNumber,
-        labels: mergeConflictLabel}));
-      await context.github.issues.createComment(params);
-    }
+  // eslint-disable-next-line no-console
+  console.log(' IS PR MERGEABLE AFTER POLLING? ' + pullRequest.mergeable);
 
-    if (hasMergeConflictLabel === true && isMergeable === true) {
-      await context.github.issues.removeLabel(context.repo({
-        number: pullRequestNumber,
-        name: mergeConflictLabel[0]
-      }));
+  var labels = pullRequest.labels;
+  for (var label in labels) {
+    if (labels[label].name === mergeConflictLabel[0]) {
+      hasMergeConflictLabel = true;
+      break;
     }
   }
+
+  var isMergeable = pullRequest.mergeable;
+
+  if (hasMergeConflictLabel === false && isMergeable === false) {
+    var userName = pullRequest.user.login;
+    var linkText = 'link';
+    var linkResult = linkText.link(
+      'https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/');
+    var params = context.repo({
+      number: pullRequestNumber,
+      body: 'Hi @' + userName +
+          '. Due to recent changes in the "develop" branch, ' +
+          'this PR now has a merge conflict. ' +
+          'Please follow this ' + linkResult +
+          ' if you need help resolving the conflict, ' +
+          'so that the PR can be merged. Thanks!'});
+    await context.github.issues.addLabels(context.repo({
+      number: pullRequestNumber,
+      labels: mergeConflictLabel}));
+    await context.github.issues.createComment(params);
+  }
+
+  if (hasMergeConflictLabel === true && isMergeable === true) {
+    await context.github.issues.removeLabel(context.repo({
+      number: pullRequestNumber,
+      name: mergeConflictLabel[0]
+    }));
+  }
+};
+
+module.exports.checkMergeConflictsInAllPullRequests = async function(context) {
+  var pullRequestsPromiseObj = await context.github.pullRequests.getAll(
+    context.repo({per_page: 60}));
```

Comment:
What's the meaning of "60" here? Will this behave correctly if there are more than 60 PRs?

Reply:
Added comments to explain this, as discussed. Thanks!